internal/pkg/vm/internal/vmcontext/invocation_context.go

```diff
@@ -408,7 +409,7 @@ func (r returnWrapper) Into(o specsruntime.CBORUnmarshaler) error {
 	// TODO: if inner is also a specsruntime.CBORUnmarshaler, overwrite o with inner.
```

Comment:
Link to PR that last changed this logic: https://github.com/filecoin-project/go-filecoin/pull/3833/files#diff-ac24da57d5301efa8c681bbe720aa2f4R282-R294